monitoring-bot/internal/monitor.go

```diff
@@ -93,10 +94,35 @@ func NewMonitor(envPath string, jsonPath string) (Monitor, error) {
 	}
 
 	mon.mnemonics = map[network]string{}
-	mon.mnemonics[devNetwork] = mon.env.devMnemonic
-	mon.mnemonics[testNetwork] = mon.env.testMnemonic
-	mon.mnemonics[qaNetwork] = mon.env.qaMnemonic
-	mon.mnemonics[mainNetwork] = mon.env.mainMnemonic
+	err = validateMnemonic(mon.env.devMnemonic, "devNetwork")
+	if err != nil {
+		return mon, err
+	} else {
```

Comment:
remove the else, if we are done with the if in 98-100 we are guaranteed there's no error
```
err = validate...
if err != nil {
  return mon, err
}
mon.mnemonics[devNetwork] = ...
```